src/utils.ts

```diff
@@ -394,4 +394,26 @@ export const getOpenapiType = async function(configFile: string):Promise<string>
     return types.indexOf(type) !== -1 ;
   }
   
+}
+
+interface LintVersion {
+  classic:string
+  present:string
+}
+
+export const getLinterVersion = function ():LintVersion {
+  let classicLintVersion =  process.env['CLASSIC_LINT_VERSION'] 
```

Comment:
If the vairbale not changed, use const if better. BTW, you can use arrow function instead assign function to a variable.

Reply:
here , the variables will be changed